pkgs/tools/networking/i2p/default.nix

```diff
@@ -36,6 +35,7 @@ stdenv.mkDerivation rec {
     description = "Applications and router for I2P, anonymity over the Internet";
     maintainers = [ stdenv.lib.maintainers.joelmo ];
     licenses = licenses.gpl2;
-    platforms = platforms.linux;
+    # TODO: support other systems, just copy appropriate lib/wrapper.. to $out
+    platforms = [ "x86_64-linux" ];
```

Comment:
But before this the check above was not to build for x86_64-linux, are you sure about this change?